apps/www/.vitepress/config.mts

```diff
@@ -70,6 +70,13 @@ export default defineConfig({
   rewrites: {
     'content/(.*)': '(.*)',
   },
+  vue: {
+    template: {
+      compilerOptions: {
+        isCustomElement: tag => tag === 'option.icon',
+      },
```

Comment:
I think there is no need for it cause we don't use `Web Components` in this project

If you want to use `option.icon` use it like

```html
<component :is="option.icon" />
```

https://vuejs.org/guide/essentials/component-basics#dynamic-components

Reply:
I was receiving vue/core warning in devtools console, so added this lines for IDE and vue/core fix ☺️

Reply:
Yes. Change `<option.icon>` to `<component :is="option.icon" />` will render the icon nicely. We dont need custom element here.

Reply:
Changed in this commit https://github.com/radix-vue/shadcn-vue/pull/334/commits/b09910fb2eb6f6dbad20a4cb6971cb6f17f5f4c7